Update-SSHKeys.md

```diff
@@ -0,0 +1,22 @@
+# Updating SSH Keys with the MCD
+
+## Background
+
+The installer creates a single user "core" with SSH keys specified at install time.  The MCD has new functionality to allow users of OpenShift 4.0 to update the SSH keys for this user "core" via a machine config's Ignition section.
+
+## What we can do
+The MCD is able to update the SSHKeys for the user "core" via a machineconfig. The keys are updated for all members of the machineconfig pool specified in the machineconfig - for example: all worker nodes.
+
```

Comment:
Maybe an example machineconfig of how it looks?

Reply:
Agreed :+1:

Reply:
@jlebon Yep this is half finished, but it will 100% have a machineconfig! I intend to make it easy so that people know exactly what to do. :+1: